package.json

```diff
@@ -12,7 +12,8 @@
     "clear": "docusaurus clear",
     "serve": "docusaurus serve",
     "write-translations": "docusaurus write-translations",
-    "write-heading-ids": "docusaurus write-heading-ids"
+    "write-heading-ids": "docusaurus write-heading-ids",
+    "to-pdf": "./convert-to-pdf.sh"
```

Comment:
I tried `"to-pdf": "yarn build && yarn serve && ./convert-to-pdf.sh"` but after serving the application, the terminal was occupied so `./convert-to-pdf.sh` didn't execute. 

I also tried using [concurrently](https://www.npmjs.com/package/concurrently) to run `yarn serve` and `./convert-to-pdf` at the same time. 
```json 
"serve-convert": "yarn build && concurrently --prefix-length 0 \"yarn serve\" \"yarn to-pdf\""
```
Though it worked, the console output was bit messy, I did try to remove the pre-fix but it is the default output of `concurrently`
![image](https://user-images.githubusercontent.com/58532267/140651704-38debbee-bb0b-4b6e-871a-202a7c1ab93d.png)

Do you have any suggestion how to `serve` and run the shell script?

Reply:
Another way to do this would be to write a node script that starts a static web server to host the build dir, and when it's done, calls your shell script in a child process, and when that's done, closes the server and quits.